frontend/public/components/build-config.tsx

```diff
@@ -148,11 +148,11 @@ const buildStrategy = (buildConfig: K8sResourceKind): BuildStrategyType =>
   buildConfig.spec.strategy.type;
 
 const allStrategies = [
-  BuildStrategyType.Docker,
-  BuildStrategyType.Devfile,
-  BuildStrategyType.JenkinsPipeline,
-  BuildStrategyType.Source,
-  BuildStrategyType.Custom,
+  { id: BuildStrategyType.Docker, title: i18n.t('public~Docker') },
+  { id: BuildStrategyType.Devfile, title: i18n.t('public~Devfile') },
+  { id: BuildStrategyType.JenkinsPipeline, title: i18n.t('public~JenkinsPipeline') },
+  { id: BuildStrategyType.Source, title: i18n.t('public~Source') },
+  { id: BuildStrategyType.Custom, title: i18n.t('public~Custom') },
 ];
```

Comment:
Need to move this into `BuildConfigsPage` -- don't want to be doing `i18n.t` in the root of a file. If we do that, the i18n logic fires once at the first import and gets cached. The user changing their language would then see the old language here as it was cached.